src/encode.rs

```diff
@@ -27,6 +32,10 @@ pub trait EncodeAsMap: Encode {
 
 #[impl_for_tuples(1, 10)]
 impl Encode for Tuple {
+    fn is_empty(&self) -> bool {
+        for_tuples!( #( Tuple.is_empty() )&* );
```

Comment:
Might make sense opening a "good first issue" for increasing code coverage, since it would be straightforward, just a bit tiresome :-)